gensim/models/wrappers/ldamallet.py

```diff
@@ -268,11 +271,16 @@ def train(self, corpus):
         cmd = self.mallet_path + ' train-topics --input %s --num-topics %s  --alpha %s --optimize-interval %s '\
             '--num-threads %s --output-state %s --output-doc-topics %s --output-topic-keys %s '\
             '--num-iterations %s --inferencer-filename %s --doc-topics-threshold %s'
+
+        if self.random_seed != None:
```

Comment:
simply `if self.random_seed` enough

Reply:
Reason for None is so that random seed is not invoked unless a value is explicitly passed, which can be zero. If I test just for if self.random_seed and zero is passed, will it enter this if - a zero is a valid random_seed value.

Reply:
so, in this case, should be `if self.random_seed is not None` (see [best practicies](https://www.python.org/dev/peps/pep-0008/#programming-recommendations))